local_server/auth/auth_test.py

```diff
@@ -0,0 +1,73 @@
+from local_server.auth.auth import AuthTypeClientBase, AuthTypeFactory
+from flask import session, request, redirect
+
+
+class AuthTypeTest(AuthTypeClientBase):
```

Comment:
Why is this code in the main source tree, and enabled for use via the config (rather than being part of the test framework only)?  It seems prone to possible issues if the user accidentally configures the test auth adaptor.

Reply:
Alternatively, the registration of the test adaptor could be predicated on some other signal that it is actually being used in a test environment.  I'd just like to see it completely disabled in PROD environments for safety.

Reply:
So it might be tough to detect if we're in a prod environment at that point in the code, but I threw together a [commit](https://github.com/chanzuckerberg/cellxgene/pull/2052/commits/f1aa655adcd0bca4684583b347b85338c0ce8b97) with an "insecure_test_environment" config setting that must be set to true or it will refuse to use AuthTypeTest. Does that help?